e2e-tests/gef/cypress/integration/unissued-facilities-AIN.spec.js

```diff
@@ -0,0 +1,523 @@
+    });
+
+    it('change links should appear for facility four and three should be unissued still', () => {
+      const issuedDate = format(oneMonth, 'd MMMM yyyy');
+      const coverStart = format(twoMonths, 'd MMMM yyyy');
+      const coverEnd = format(threeMonthsOneDay, 'd MMMM yyyy');
+
+      applicationPreview.reviewFacilityStage().contains('Review facility stage');
+      applicationPreview.updatedUnissuedFacilitiesHeader().contains('The following facility stages have been updated to issued:');
+      applicationPreview.updatedUnissuedFacilitiesList().contains(unissuedFacilitiesArray[0].name);
+      applicationPreview.updatedUnissuedFacilitiesList().contains(`${unissuedFacilitiesArray[2].name}name`);
+
+      applicationPreview.facilitySummaryListRowValue(0, 0).contains(`${unissuedFacilitiesArray[2].name}name`);
+      applicationPreview.facilitySummaryListRowAction(0, 0).contains('Change');
+      applicationPreview.facilitySummaryListRowAction(0, 1).should('have.value', '');
+      applicationPreview.facilitySummaryListRowValue(0, 2).contains('Issued');
+      applicationPreview.facilitySummaryListRowAction(0, 2).should('have.value', '');
+      applicationPreview.facilitySummaryListRowValue(0, 3).contains(issuedDate);
+      applicationPreview.facilitySummaryListRowAction(0, 3).contains('Change');
+      applicationPreview.facilitySummaryListRowValue(0, 4).contains(coverStart);
+      applicationPreview.facilitySummaryListRowAction(0, 4).contains('Change');
+      applicationPreview.facilitySummaryListRowValue(0, 5).contains(coverEnd);
+      applicationPreview.facilitySummaryListRowAction(0, 5).contains('Change');
+
+      applicationPreview.facilitySummaryListRowValue(1, 0).contains(MOCK_FACILITY_THREE.name);
+      applicationPreview.facilitySummaryListRowAction(1, 0).should('have.value', '');
+      applicationPreview.facilitySummaryListRowAction(1, 1).should('have.value', '');
+      applicationPreview.facilitySummaryListRowValue(1, 2).contains('Unissued');
+      applicationPreview.facilitySummaryListRowAction(1, 2).contains('Change');
+      applicationPreview.facilitySummaryListRowKey(1, 2).should('not.have.value', 'Date issued to exporter');
+      applicationPreview.facilitySummaryListRowKey(1, 3).should('not.have.value', 'Cover start date');
+      applicationPreview.facilitySummaryListRowKey(1, 4).should('not.have.value', 'Cover end date');
+    });
+
+    it('pressing submit button takes you to submit page and with correct panel once submitted to checker', () => {
+      applicationPreview.submitButtonPostApproval().click();
+      applicationSubmission.submissionText().contains('Someone at your bank must check your update before they can submit it to UKEF');
+      applicationSubmission.submitButton().click();
+      applicationSubmission.confirmationPanelTitleFacilities().contains('Issued facilities submitted for checking at your bank');
```

Comment:
Can we assert the URL and for any other tests in here that have URL redirection?

Reply:
done